src/main/java/me/totalfreedom/totalfreedommod/caging/Cager.java

```diff
@@ -96,6 +97,18 @@ public void onPlayerQuit(PlayerQuitEvent event)
     }
 
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
+    public void onPlayerKick(PlayerKickEvent event)
```

Comment:
Why ignoreCancelled? If the event is cancelled, why do you need to remove the cage?